components/ScheduleCard.js

```diff
@@ -21,6 +30,19 @@ export default class ScheduleCard extends Component<Props> {
             position: "absolute", alignSelf: "flex-end",
             flex: 1, padding: 8
           }}>
+            <Text>Subscribe to Event?</Text>
+            <Switch
+            value={this.state.switchValue}
+            onValueChange ={(switchValue)=> {
+              this.setState({switchValue});
+              if (switchValue) OneSignal.sendTag(this.props.title + "eventid", "subscribed");
+              else OneSignal.deleteTag(this.props.title + "eventid");
+              OneSignal.getTags((receivedTags) => {
+                  console.log(receivedTags);
```

Comment:
remove this at least - i'd prefer you replace this without onesignal as discussed above